modules/bar/media/index.ts

```diff
@@ -33,8 +33,8 @@ const Media = () => {
 
     const songIcon = Variable("");
 
-    const mediaLabel = Utils.watch("󰎇 Media 󰎇", [mpris, show_artist, truncation, truncation_size], () => {
-        if (activePlayer.value) {
+    const mediaLabel = Utils.watch("󰎇 Media 󰎇", [mpris, show_artist, truncation, truncation_size, show_label], () => {
```

Comment:
```suggestion
    const mediaLabel = Utils.watch("Media", [mpris, show_artist, truncation, truncation_size, show_label], () => {
```
As mentioned below, we'll leave this to the widget that draws the icons rather than drawing it in the label itself. I should have done it this way initially so let's fix it while we're at it.